packages/@angular/cli/blueprints/ng2/files/__path__/app/app.component.spec.ts

```diff
@@ -18,21 +18,21 @@ describe('AppComponent', () => {
   });
 
   it('should create the app', async(() => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.debugElement.componentInstance;
+    const fixture: ComponentFixture<AppComponent> = TestBed.createComponent(AppComponent);
```

Comment:
The structures of the two files are different.  I think harmonizing the structures of the two component specs might be something to consider.
Either way, typescript is quite good at inferring types.  In fact, TS 2.1+ (with noImplicitAny enabled) can actually infer the type shown in the linked component template.